includes/class-wc-swedbank-plugin.php

```diff
@@ -0,0 +1,527 @@
+
+			if ( isset( $gateways[ $payment_method ] ) ) {
+				/** @var \WC_Gateway_Swedbank_Pay_Cc $gateway */
+				$gateway = $gateways[ $payment_method ];
+
+				wc_get_template(
+					'admin/action-buttons.php',
+					[
+						'gateway' => $gateway,
+						'order'   => $order
+					],
+					'',
+					dirname( __FILE__ ) . '/../templates/'
+				);
+			}
+		}
+	}
+
+	/**
+	 * Enqueue Scripts in admin
+	 *
+	 * @param $hook
+	 *
+	 * @return void
+	 */
+	public static function admin_enqueue_scripts( $hook ) {
+		if ( $hook === 'post.php' ) {
+			// Scripts
+			$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+			wp_register_script( 'swedbank-pay-admin-js',
+				plugin_dir_url( __FILE__ ) . '../assets/js/admin' . $suffix . '.js' );
+
+			// Localize the script
+			$translation_array = [
+				'ajax_url'  => admin_url( 'admin-ajax.php' ),
+				'text_wait' => __( 'Please wait...', 'swedbank-pay-woocommerce-payments' ),
+			];
+			wp_localize_script( 'swedbank-pay-admin-js', 'SwedbankPay_Admin', $translation_array );
+
+			// Enqueued script with localized data
+			wp_enqueue_script( 'swedbank-pay-admin-js' );
+		}
+	}
+
+	/**
+	 * Action for Capture
+	 */
+	public function ajax_swedbank_pay_capture() {
+		if ( ! wp_verify_nonce( $_REQUEST['nonce'], 'swedbank_pay' ) ) {
+			exit( 'No naughty business' );
+		}
+
+		$order_id = (int) $_REQUEST['order_id'];
+		$order    = wc_get_order( $order_id );
+
+		// Get Payment Gateway
+		$gateways = WC()->payment_gateways()->get_available_payment_gateways();
+		if ( isset( $gateways[ $order->get_payment_method() ] ) ) {
+			$gateway = $gateways[ $order->get_payment_method() ];
+
+			try {
+				$gateway->capture_payment( $order_id );
+				wp_send_json_success( __( 'Capture success.', 'swedbank-pay-woocommerce-payments' ) );
+			} catch ( Exception $e ) {
+				$message = $e->getMessage();
+				wp_send_json_error( $message );
+			}
+		}
+	}
+
+	/**
+	 * Action for Cancel
+	 */
+	public function ajax_swedbank_pay_cancel() {
+		if ( ! wp_verify_nonce( $_REQUEST['nonce'], 'swedbank_pay' ) ) {
+			exit( 'No naughty business' );
+		}
+
+		$order_id = (int) $_REQUEST['order_id'];
+		$order    = wc_get_order( $order_id );
+
+		// Get Payment Gateway
+		$gateways = WC()->payment_gateways()->get_available_payment_gateways();
+		if ( isset( $gateways[ $order->get_payment_method() ] ) ) {
+			$gateway = $gateways[ $order->get_payment_method() ];
+
+			try {
+				$gateway->cancel_payment( $order_id );
+				wp_send_json_success( __( 'Cancel success.', 'swedbank-pay-woocommerce-payments' ) );
+			} catch ( Exception $e ) {
+				$message = $e->getMessage();
+				wp_send_json_error( $message );
+			}
+		}
+	}
+
+	/**
+	 * Generate UUID
+	 *
+	 * @param $node
+	 *
+	 * @return string
+	 */
+	public function generate_uuid( $node ) {
+		return \Ramsey\Uuid\Uuid::uuid5( \Ramsey\Uuid\Uuid::NAMESPACE_OID, $node )->toString();
+	}
+
+	/**
+	 * Dispatch Background Process
+	 */
+	public function maybe_process_queue() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)